### ISSUE TITLE
[SECURITY] Backdoored dependencies 

### ISSUE THREAD

**Issue by dbendaou** [Semantic-Org/Semantic-UI#6687]
Hi guys, 
Don't know if you've heard about it, but we really need to double check before adding any dependencies on the repo. 

One of them as been corrupted : 
<img width="341" alt="capture d ecran 2018-11-27 a 14 18 30" src="https://user-images.githubusercontent.com/15218861/49084369-902ce680-f24f-11e8-98ec-36e41506699b.png">

Here is the issue on the original repo --> https://github.com/dominictarr/event-stream/issues/116

It's event-stream that has been corrupted with a dependencies named `flatmap-stream`

**Comment by dbendaou** [comment]
I fully agree with you ! I was just about to share the information with you guys, and since we use `event-stream` it was kind of our concern too !

**Comment by hammy2899** [comment]
@championswimmer I am aware that it has been removed from npm but it is still an issue for users who have the version cached. Developers have been advised to bump the version to `event-stream@3.3.4` to prevent this.
